File: 242/242-easy.go
package main

import (
	"fmt"
	"os"
	"strconv"
)

func check(e error) {
	if e != nil {
		fmt.Println(e)
		os.Exit(2)
	}
}

func plant(people, plants int) int {
	var weeks, fruits int
	for fruits < people {
		weeks++
		fruits += plants
		plants += fruits
	}
	return weeks + 1
}

func main() {
	args := os.Args[1:]
	people, err := strconv.Atoi(args[0])
	check(err)
	plants, err := strconv.Atoi(args[1])
	check(err)
	weeks := plant(people, plants)
	fmt.Println(weeks)
}

File: 242/242-easy.js
const input = process.argv.slice(2);
const people = parseInt(input[0], 10);
const plants = parseInt(input[1], 10);

function plant (people, plants) {
  'use strict';
  let weeks = 0;
  let fruits = 0;
  while (fruits < people) {
    weeks++;
    fruits += plants;
    plants += fruits;
  }
  return weeks + 1;
}

console.log(plant(people, plants));
